pywinauto/uia_element_info.py

```diff
@@ -259,21 +238,19 @@ def _get_elements(self, tree_scope, cond = IUIA().true_condition, cache_enable =
         return elements_from_uia_array(ptrs_array, cache_enable)
 
     def children(self, **kwargs):
-        """
-        Return a list of only immediate children of the element
-        
-        * **kwargs** is a criteria to reduce a list by process, 
+        """Return a list of only immediate children of the element
+
+        * **kwargs** is a criteria to reduce a list by process,
         class_name and/or title.
```

Comment:
Add to description recently added filters: 'control_type' and 'is_content_element':
" ... class_name, control_type, is_content_element and/or title."

The same for descendants method doc string too.